extensions/sql/processors/ExecuteSQL.h

```diff
@@ -42,7 +43,8 @@ class ExecuteSQL : public SQLProcessor, public FlowFileSource {
   }
 
   EXTENSIONAPI static const core::Relationship Success;
-  static auto relationships() { return std::array{Success}; }
+  EXTENSIONAPI static const core::Relationship Failure;
+  static auto relationships() { return std::array{Success, Failure}; }
```

Comment:
please add these two new `failure` relationships to `PROCESSORS.md`, too

Reply:
added